### PR TITLE
Add LRS3 data preparation

### PR DESCRIPTION
**Pull request by mpc001**
This PR adds a data preparation recipe that uses the ultra face detector to extract full-face video. The resulting video output is then used as input for training and evaluating RNNT-based models for automatic speech recognition (ASR), visual speech recognition (VSR), and audio-visual ASR (AV-ASR) on the LRS3 dataset.

This PR also updates the word error rate (WER) for AV-ASR LRS3 models and improves the code readability.

### CODE REVIEW

File: examples/asr/avsr_rnnt/data_prep/main.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
+import glob
+import math
+import os
+import shutil
+
+import warnings
+
+import ffmpeg
```
Comment (mthrok) [inline examples/asr/avsr_rnnt/data_prep/main.py:8]: This one is not something I am familiar with. Is there a direction on how to install this in README or requirements.txt? if not, can you add it?

**Review by mthrok — APPROVED** [review]
The code looks fine, but IIUC, the preparation uses four different FFmpeg-related packages. torchvision, opencv, ffmpeg, torchaudio.

This might cause some subtle difference in the data processing.

File: examples/asr/avsr_rnnt/data_prep/main.py (hunk 1 above)
Comment (mpc001) [reply examples/asr/avsr_rnnt/data_prep/main.py:8]: Thanks for pointing out this. `ffmpeg-python` has been added to requirements.txt

File: examples/asr/avsr_rnnt/data_prep/main.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+from utils import save_vid_aud_txt, split_file
+
+warnings.filterwarnings("ignore")
+
+from argparse import ArgumentParser
+
+
+def load_args(default_config=None):
+    parser = ArgumentParser()
+    # -- for benchmark evaluation
```
[the remark this answers is not in the document]
Comment (mpc001) [reply examples/asr/avsr_rnnt/data_prep/main.py]: Sure, I have included `description`.

**Pull request closed without merging**